db/post_migrate/20220429101850_clear_email_domain_blocks.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ClearEmailDomainBlocks < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  class EmailDomainBlock < ApplicationRecord
+  end
+
+  def up
+    EmailDomainBlock.where.not(parent_id: nil).in_batches.delete_all
+  end
+
+  def down; end
+end
```

Comment:
I understand why you're deleting past possibly-automatically-created email domain blocks, but I'm not sure about it…